DirectX11/nvprofile.cpp

```diff
@@ -1,12 +1,12 @@
-#include "../util.h"
-//#include "Globals.h"
 #include "IniHandler.h"
 #include "D3D11Wrapper.h"
 
 // Including this after the above headers so the nvapi.h include will pick up
 // the d3d11.h pre-processor defines to avoid mis-matches with profiling.h:
 #include "nvprofile.h"
 
+#include <util.h>
+
```

Comment:
I'm sensing there might have been some circular include conflicts to require this header reorganisation? We should perhaps consider moving to a model where header files prefer to use forward declarations over including other headers where possible, and the cpp files include the headers they need, as that can help a lot to avoid the circular include dependency hell.